Android.mk

```diff
@@ -590,16 +590,16 @@ ifeq ($(TARGET_USE_TRUSTY),true)
 endif
 LOCAL_STATIC_LIBRARIES += libavb_kernelflinger-$(TARGET_BUILD_VARIANT)
 
-keys4aic_intermediates := $(call intermediates-dir-for,EFI,keys)
+keys4cic_intermediates := $(call intermediates-dir-for,EFI,keys)
 
-AIC_VERITY_CERT := $(keys4aic_intermediates)/verity.cer
-AIC_PADDED_VERITY_CERT := $(keys4aic_intermediates)/verity.padded.cer
-AIC_OEMCERT_OBJ := $(keys4aic_intermediates)/oemcert.o
+CIC_VERITY_CERT := $(keys4cic_intermediates)/verity.cer
```

Comment:
I know it has nothing to do with this patch goal but I see there are more and more duplication in this makefile.  You should try to factorize.  For instance, you could create and use Makefile functions.